### PR TITLE
Provide pre-compiled bundle.

### PR DESCRIPTION
**Pull request by busykoala**
There is still an error:
```
jquery.min.js:5 GET http://localhost:8080/plonejsi18n?domain=widgets&language=de 404 (Not Found)
```
I'm not sure if the error is relevant at all, but it's there.

TODO:
- [ ] Fix jquery error
- [x] Add upgrade step

### CODE REVIEW

**Comment by busykoala** [comment]
@4teamwork/plone I need help fixing the error described in the header comment.

**Comment by jone** [comment]
Is `plonejsi18n` something of ftw.book or of Plone?
Is this a Book problem?

**Comment by busykoala** [comment]
I assume that it's from Plone since I do not register anything alike. Still it wouldn't occur in plain Plone I think 🤔

**Comment by djowett-ftw** [comment]
It seems to be made available in  (or near to)  `parts/omelette/mockup/js/i18n.js` therefore I would say that error is not a good sign.  i.e. it's something that should be working going forwards.

**Comment by jone** [comment]
I don't have the `plonejsi18n` on a fresh installation. I don't think that `ftw.book` has anything to do with that problem. I say we move forward.

**Comment by busykoala** [comment]
> I don't have the `plonejsi18n` on a fresh installation. I don't think that `ftw.book` has anything to do with that problem. I say we move forward.

Ok, then lets risk it and go with the js we have now. I added a condition to your upgrade step so `registry.xml` is only upgraded in plone5.
@jone, could you please check out the pull again?

File: ftw/book/upgrades/20191216174215_update_resource_bundle/upgrade.py [hunk 1]
```diff
@@ -6,4 +7,8 @@ class UpdateResourceBundle(UpgradeStep):
     """
 
     def __call__(self):
-        self.install_upgrade_profile()
+        if IS_PLONE_5:
+            self.install_upgrade_profile()
+        else:
```
Comment (Nachtalb) [inline ftw/book/upgrades/20191216174215_update_resource_bundle/upgrade.py]: You don't need the else part

Comment (busykoala) [reply ftw/book/upgrades/20191216174215_update_resource_bundle/upgrade.py]: I kind of like to have it there to emphasise on the fact that in plone4 this should not be done and explicitly comment that. But I agree that it is pretty useless in means of it not doing anything.

Comment (Nachtalb) [reply ftw/book/upgrades/20191216174215_update_resource_bundle/upgrade.py]: Why not just put the comment above the "if" then XD Something like this.
```
# The registry entries only have to be installed on plone 5
if IS_PLONE_5:
   ...
```

Comment (busykoala) [reply ftw/book/upgrades/20191216174215_update_resource_bundle/upgrade.py]: That a good approach, thanks for the advise. I changed it in 1380d0c

**Pull request merged**